server/instances.go

```diff
@@ -173,7 +174,14 @@ func (p *Plugin) UninstallInstance(instanceID types.ID, instanceType InstanceTyp
 			var err error
 			instance, err = p.instanceStore.LoadInstance(instanceID)
 			if err != nil {
-				return err
+				if strings.Contains(err.Error(), "not found") {
+					instances.Delete(instanceID)
+					updated = instances
+					if err = p.instanceStore.StoreInstances(instances); err != nil {
+						return err
+					}
+				}
+				return nil
 			}
```

Comment:
```suggestion
				if !strings.Contains(err.Error(), "not found") {
					return err
				}
				
				instances.Delete(instanceID)
				updated = instances
				if err = p.instanceStore.StoreInstances(instances); err != nil {
					return err
				}

				return nil
			}
```

Reply:
What does updating the `updated` variable do here since we return afterwards?

Do we need to perform any of the operations that are in the rest of this function `UninstallInstance`?

Reply:
.> Also, what does updating the `updated` variable do here since we return afterwards?
    Removed the updated variable 
> Do we need to perform any of the operations that are in the rest of this function `UninstallInstance`?
The rest of the function does only one thing which is disconnecting the users connected to that instance. If the instance is not found there is very unlikely that users are connected to that instance. But we can still check whether the users are connected if connected then we can disconnect them.

Reply:
@Kshitij-Katiyar Do we have a theory of how the kv store got into this state in the first place? If not, I think we should do as much cleanup as possible here. No reason to leave behind orphaned records such as the user's connected info

Reply:
> Removed the updated variable

Looks like it's still there